From 9165bfb534c01e49ca01304181dfbf914e8aac8c Mon Sep 17 00:00:00 2001
From: Ivan Delalande <colona@arista.com>
Date: Wed, 24 Jan 2018 17:16:22 -0800
Subject: [PATCH] lkdtm: fix handle_irq_event symbol for INT_HW_IRQ_EN

References: bsc#1052766
Patch-mainline: v4.16-rc1
Git-commit: 5be2a5011c039506e2862650c928acfb2e3d7b9c

Function handle_IRQ_event was retired in v2.6.39 and replaced with
handle_irq_event but nobody changed it in lkdtm so INT_HW_IRQ_EN has
been broken for a while.

Fixes: 33b054b867b8 ("genirq: Remove handle_IRQ_event")
Signed-off-by: Travis Brown <travisb@arista.com>
Signed-off-by: Ivan Delalande <colona@arista.com>
Acked-by: Kees Cook <keescook@chromium.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Acked-by: Michal Suchanek <msuchanek@suse.de>
---
 drivers/misc/lkdtm_core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/misc/lkdtm_core.c b/drivers/misc/lkdtm_core.c
index 42d2b8e31e6b..930b30c64880 100644
--- a/drivers/misc/lkdtm_core.c
+++ b/drivers/misc/lkdtm_core.c
@@ -158,7 +158,7 @@ struct crashpoint crashpoints[] = {
 	CRASHPOINT("INT_HARDWARE_ENTRY",	lkdtm_debugfs_entry,
 		   "do_IRQ",			jp_do_irq),
 	CRASHPOINT("INT_HW_IRQ_EN",		lkdtm_debugfs_entry,
-		   "handle_IRQ_event",		jp_handle_irq_event),
+		   "handle_irq_event",		jp_handle_irq_event),
 	CRASHPOINT("INT_TASKLET_ENTRY",		lkdtm_debugfs_entry,
 		   "tasklet_action",		jp_tasklet_action),
 	CRASHPOINT("FS_DEVRW",			lkdtm_debugfs_entry,
-- 
2.13.6

